src/lib/core/components/computations/plugins/abundance.tsx

```diff
@@ -63,12 +64,16 @@ export function AbundanceConfiguration(props: ComputationConfigProps) {
           value={collectionVariable}
           onChange={(e) => setCollectionVariable(e.target.value)}
         >
-          {ABUNDANCE_COLLECTION_VARIABLES.map((collectionVar) => {
-            const result = findEntityAndVariable(entities, collectionVar);
+          {collections.map((collectionVar) => {
             return (
-              result && (
-                <option value={variableDescriptorToString(collectionVar)}>
-                  {result.entity.displayName}: {result.variable.displayName}
+              collectionVar && (
+                <option
+                  value={variableDescriptorToString({
+                    variableId: collectionVar.id,
+                    entityId: collectionVar.entityId,
+                  })}
+                >
+                  {collectionVar.entityDisplayName}: {collectionVar.displayName}
                 </option>
               )
```

Comment:
I think this can be changed to the following, once the return-value of the callback passed to `flatMap` in `findCollectionVariables` is updated as I suggested.
```suggestion
                <option
                  value={variableDescriptorToString({
                    variableId: collectionVar.id,
                    entityId: collectionVar.entityId,
                  })}
                >
                  {collectionVar.entityDisplayName}: {collectionVar.displayName}
                </option>
```